pkg/deploy/controller/configchange/controller.go

```diff
@@ -110,36 +112,10 @@ func (c *DeploymentConfigChangeController) generateDeployment(config *deployapi.
 	// This update is atomic. If it fails because a newer resource was already persisted, that's
 	// okay - we can just ignore the update for the old resource and any changes to the more
 	// current config will be captured in future events.
-	updatedConfig, err := c.changeStrategy.updateDeploymentConfig(config.Namespace, newConfig)
+	updatedConfig, err := c.client.DeploymentConfigs(config.Namespace).UpdateStatus(newConfig)
```

Comment:
@ironcladlou can you confirm that the configchange controller is just updating the status of the deploymentconfig? I don't see anything in the generator touching the spec.